app/models/concerns/netlify_content.rb

```diff
@@ -23,6 +23,16 @@ def find_by_path(path, base:, try_index:)
       end
     end
 
+    def find_by_slug(slug, base:)
```

Comment:
[question] What's the benefit of using this method over `find_by_path` with `try_index: false`?

Reply:
Netlify tends to think in slugs for relations. I think there is a benefit to reusing some of that terminology in our code, but I get what you mentioned above about the conflict with `Page.find_by_slug`. Maybe `find_by_slug` could be an alias for `find_by_path try_index: false`?

Reply:
Actually going back and looking at the code, I agree with you.